src/normalize/utils.js

```diff
@@ -77,11 +93,122 @@ export const forceActivitiesWithStrictIncreasingDateCreated = activities => {
   })
 }
 
-export const sortedHydratedActivitiesFrom = (state, activities) => {
+export const sortedHydratedActivitiesFrom = activities => {
   const hydratedSortedActivities = (activities || []).map(hydratedActivityFrom)
   hydratedSortedActivities.sort((activity1, activity2) =>
     new Date(activity1.dateCreated) < new Date(activity2.dateCreated) ? -1 : 1
   )
   forceActivitiesWithStrictIncreasingDateCreated(hydratedSortedActivities)
   return hydratedSortedActivities
 }
+
+export const deletionHelpersFrom = (
+  state,
+  activities,
+  stateKeysByEntityIdentifier
+) => {
+  const deletedActivityIdentifiers = []
+  const deletedActivityIdentifiersByStateKey = {}
+  activities
+    .filter(
+      activity =>
+        activity.verb === 'delete' ||
+        (activity.patch && activity.patch.isSoftDeleted)
```

Comment:
C'est pas le symptôme d'un manque d'homogénéité ?

Reply:
les activités doivent faire disparaite à la fois les entites qu'on peut soft deleter ET celle qu'on peut deleter direct, donc j'ai as l'impression qu'il y a un manque ici ?